src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageLoggerExtensions.cs

```diff
@@ -41,6 +42,10 @@ static PageLoggerExtensions()
                3,
                "Request was short circuited at page filter '{PageFilter}'.");
 
+            _redirectToPageResultExecuting = LoggerMessage.Define<string>(
+                LogLevel.Information,
+                1,
```

Comment:
I messed up the copy-pasta. @ryanbrandenburg filed a bug to make sure we aren't messing up elsewhere.